.gitignore

```diff
@@ -23,3 +23,4 @@ jenkins*home/
 jenkins-plugins/
 .jenkins_test/
 !src/test/resources/jacoco/test/target
+jut.sh
```

Comment:
@stefanSchuhbaeck @koceluch 
Can you please remove the .gitignore changes so we can merge this change?